docs/tutorials/aws.md

```diff
@@ -80,21 +93,60 @@ spec:
     spec:
       containers:
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.2
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.7
         args:
         - --source=service
         - --source=ingress
         - --domain-filter=external-dns-test.my-org.com # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
         - --provider=aws
         - --policy=upsert-only # would prevent ExternalDNS from deleting any records, omit to enable full synchronization
+        - --aws-zone-type=public #only look at public hosted zones (valid values are public/private)
         - --registry=txt
         - --txt-owner-id=my-identifier
 ```
 
-## Verify ExternalDNS works
+## Arguments
+
+This list is not the full list, but a few arguments that where choosen.
+
+### aws-zone-type
+
+`aws-zone-type` allows filtering for private and public zones
```

Comment:
Can we put this at the end of the tuturial? The flow should be a quick and easy wait to get started (with mostly using the default flag values) and then have a section at the end for all kinds of (optional) configuration (like filtereing for zone types etc.)

Reply:
/cc @shavo007